modules/central-services-public-config.adoc

```diff
@@ -158,6 +158,70 @@ Red Hat recommends that you do not specify a port number if you are exposing {pr
 | `central.exposure.route.enabled`
 | Use `true` to expose Central by using a route.
 This parameter is only available for {ocp} clusters.
+
+| `central.db.external`
+| *Tech Preview* Use `true` to specify that Central DB should not be deployed and that an external database will be used.
+
+| `central.db.source.connectionString`
+a| *Tech Preview* The connection string for Central to use to connect to the database. This is only used when `central.db.external` is set to true. The connection string must be in keyword/value format as described in https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING[PostgreSQL Docs]
+
+| `central.db.source.minConns`
+| The minimum number of connections to the database to be established.
+
+| `central.db.source.maxConns`
+| The maximum number of connections to the database to be established.
+
+| `central.db.source.statementTimeoutMs`
+| The number of milliseconds a single query or transaction can be active against the database.
+
+| `central.db.postgresConfig`
+| The postgresql.conf to be used for Central DB.
+
+| `central.db.hbaConfig`
+| The pg_hba.conf to be used for Central DB.
```

Comment:
Thoughts on linking to https://www.postgresql.org/docs/current/auth-pg-hba-conf.html ?